src/Entity/BaseEvent.php

```diff
@@ -101,6 +101,13 @@
      */
     protected $description;
 
+    /**
+     * @var string|null
+     *
+     * @ORM\Column(length=50)
```

Comment:
in your getter `timeZone` could be null, but not here, is there a mistake ?

Reply:
corriger